ui/src/components/StatsGraph/StatsGraph.tsx

```diff
@@ -10,11 +11,27 @@ export default function StatsGraph({ containerName, containerID }: StatsGraphPro
   // because the full container (64 digits) is too long. Grafana API sets a
   // 40 character limit on a dashboard name.
   const shortContainerID = containerID.slice(0, 12);
+  const [timeFrame, setTimeFrame] = useState('15m');
+  const handleChange = (event: SelectChangeEvent<unknown>) => {
```

Comment:
instead of defining this function here, you can pass an anonymous function to the onChange prop of the Select component.

If you do it that way, the `event` type will be inferred, so you will not need to use `SelectChangeEvent`. 

Since this is a very simple function, I think it would be an improvement to use the anonymous function.